examples/authenticated-custom-jwt/app.js

```diff
@@ -24,7 +24,7 @@ app.post("/login", (req, res) => {
     loginDetails.password === mockedPassword
   ) {
     let token = jwt.sign({ username: loginDetails.username }, config.secret, {
-      expiresIn: "24h" // expires in 24 hours
+      expiresIn: "12h" // expires in 12 hours
```

Comment:
Having this set to 24 hours resulted in an error. I think by design, but the error was no matching provider. Will investigate that behaviour.